src/mlpack/tests/cli_binding_test.cpp

```diff
@@ -13,59 +13,57 @@
 #include <mlpack/bindings/cli/cli_option.hpp>
 #include <mlpack/core/kernels/gaussian_kernel.hpp>
 
-#include <boost/test/unit_test.hpp>
-#include "test_tools.hpp"
+#include "catch.hpp"
+#include "test_catch_tools.hpp"
```

Comment:
Is this include needed here? I couldn't find usage of any function/macro from _test_catch_tools.hpp_